doc/whats-new.rst

```diff
@@ -49,10 +49,11 @@ New Features
 
 Bug fixes
 ~~~~~~~~~
+- Variables which are chunked using dask only along some dimensions can be chunked while storing with zarr along previously
+  unchunked dimensions (:pull:`4312`) `Tobias Kölling <https://github.com/d70-t>`_.
```

Comment:
nit: I know the entry below is different (not intentionally, I think), but we usually follow a certain format with these items:
```suggestion
  unchunked dimensions (:pull:`4312`) By `Tobias Kölling <https://github.com/d70-t>`_.
```